config/configauth/README.md

```diff
@@ -1,17 +1,36 @@
 # Authentication configuration for receivers
 
-This module allows server types, such as gRPC and HTTP, to be configured to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator. Currently, only bearer token authentication is supported, although the module is ready to accept new authenticators.
+This module allows server types, such as gRPC and HTTP, to be configured to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator.
+
+The currently known authenticators:
+
+- [oidc](../../extension/authoidcextension)
 
 Examples:
 ```yaml
+extensions:
+  oidc:
+    # see the blog post on securing the otelcol for information
+    # on how to setup an OIDC server and how to generate the TLS certs
+    # required for this example
+    # https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f
+    issuer_url: http://localhost:8080/auth/realms/opentelemetry
+    audience: account
+
 receivers:
-  somereceiver:
-    grpc:
-      authentication:
-        attribute: authorization
-        oidc:
-          issuer_url: https://auth.example.com/
-          issuer_ca_path: /etc/pki/tls/cert.pem
-          client_id: my-oidc-client
-          username_claim: email
+  otlp/auth:
```

Comment:
nit: name is confusion. When I read it first time it felt the auth has to be configured as a receiver.